client/src/components/CardList/CardList.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Story, Meta } from "@storybook/react";
+import React from "react";
+
+import { CardList, CardListElementProps } from "./CardList";
+
+export default {
+  title: "CardList",
+  component: CardList,
+} as Meta;
+
+type CardListProps = React.ComponentProps<typeof CardList>;
+
+const Template: Story<CardListProps> = (args) => <CardList {...args} />;
+
+const elements: CardListElementProps[] = [
+  {
+    display: "Non-link Title",
+    href: "#",
+    children: [
+      { display: "Non-link Title 1", href: "" },
+      { display: "Non-link Title 2", href: "" },
+      { display: "Non-link Title 3", href: "" },
```

Comment:
Seems like a typing mistake in the actual `CardList` props, shouldn't `href` just be optional?